src/servers/middlewares/auth-router.ts

```diff
@@ -38,9 +36,9 @@ authRouter.post("/browser", async (req, res) => {
   const logger = graphqlLogger.child({ ip, body: req.body })
 
   try {
-    const { data } = await kratos.toSession(undefined, req.header("Cookie"))
+    const { data } = await kratosPublic.toSession(undefined, req.header("Cookie"))
```

Comment:
This `toSession` method should be exposed via a `Service` - we shouldn't call the client directly. Even if its just a thin wrapper - catching and translating errors already has value.

Reply:
I didn't really touch this code (expect a function rename). This code is meant to be deleted on Cookie is activated and webwallet updated. (it was a "hack"/proxy for when there was no Oathkeeper that knew how to manage cookie)